### PR TITLE
Vote front end

### LINKED ISSUE #17: Voting page
Page containing a list of websites where players can vote for RedCraft.org

### PR DESCRIPTION
**Pull request by Cod3lta**
adding the "vote" page and reordered some components in the `rebootstrap.scss` file.
This pull request closes #17 Voting page.

### CODE REVIEW

File: website/templates/website/pages/vote.html [hunk 1]
```diff
@@ -1,8 +1,53 @@
...
+{% block content %}
+{% with title="Vote" waveFill="#202020" %}
+{% include "website/components/section-header.html" %}
+{% endwith %}
+<section id="info">
+    <div class="container">
+        <h3 class="section-titre">Voter pour le serveur</h3>
+        {% blocktranslate trimmed %}
+        <p>Sur cette page, vous trouvez plusieurs liens vous permettant de voter pour RedCraft. Ainsi, vous aidez le serveur à gagner en popularité ! Merci !</p>
+        <p>Pour l'instant, les vôtes que vous effectuez depuis cette page ne seront pas comptabilisés sur votre compte RedCraft.org.</p>
```
Comment (lululombard) [inline website/templates/website/pages/vote.html]:
```suggestion
        <p>Pour l'instant, les votes que vous effectuez depuis cette page ne seront pas comptabilisés sur votre compte RedCraft.org.</p>
```

File: website/views.py [hunk 2]
```diff
@@ -172,7 +172,33 @@ def get_context_data(self, **kwargs):
...
+                'websites_list' : [
+                    {
+                        'titre': 'Serveurs-minecraft.org',
+                        'explainations': 'Sur la page, scrollez jusqu\'au bouton vert "Voter pour RedCraft". Cliquez simplement dessus.',
+                        'url' : 'https://www.serveurs-minecraft.org/',
+                        'path_img' : 'website/img/pages/vote/serveurs-minecraft-org.png'
+                    },
+                    {
+                        'titre': 'top-serveurs.net',
+                        'explainations': 'Entrez ici une description de comment voter sur le site',
```
Comment (lululombard) [inline website/views.py]:
```suggestion
                        'explainations': 'TODO Entrez ici une description de comment voter sur le site',
```

File: website/views.py [hunk 3]
```diff
@@ -172,7 +172,33 @@ def get_context_data(self, **kwargs):
...
+                        'url' : 'https://top-serveurs.net/',
+                        'path_img' : 'website/img/pages/vote/top-serveurs.png'
+                    },
+                    {
+                        'titre': 'liste-serveur-minecraft.net',
+                        'explainations': 'Entrez ici une description de comment voter sur le site',
```
Comment (lululombard) [inline website/views.py]:
```suggestion
                        'explainations': 'TODO Entrez ici une description de comment voter sur le site',
```

File: website/views.py [hunk 4]
```diff
@@ -172,7 +172,33 @@ def get_context_data(self, **kwargs):
...
+                        'url' : 'http://www.liste-serveur-minecraft.net/',
+                        'path_img' : 'website/img/pages/vote/liste-serveurs-minecraft.png'
+                    },
+                    {
+                        'titre': 'serveursminecraft.org',
+                        'explainations': 'Entrez ici une description de comment voter sur le site',
```
Comment (lululombard) [inline website/views.py]:
```suggestion
                        'explainations': 'TODO Entrez ici une description de comment voter sur le site',
```

**Review by Likyaz — DISMISSED** [review]

File: website/views.py [hunk 5]
```diff
@@ -172,7 +172,33 @@ def get_context_data(self, **kwargs):
         return {
             **ctx,
             **{
-                'page': 'vote'
+                'page': 'vote',
+                'websites_list' : [
+                    {
+                        'titre': 'Serveurs-minecraft.org',
+                        'explainations': 'Sur la page, scrollez jusqu\'au bouton vert "Voter pour RedCraft". Cliquez simplement dessus.',
```
Comment (lululombard) [inline website/views.py]:
```suggestion
                        'explainations': 'Sur la page, descendez jusqu\'au bouton vert "Voter pour RedCraft". Cliquez simplement dessus.',
```

**Review by lululombard — CHANGES_REQUESTED** [review]
Some typo and flag TODO so we don't forget them before final release

**Review by lululombard — APPROVED** [review]

**Review by Likyaz — APPROVED** [review]

**Pull request merged**